modules/installation-vsphere-config-yaml.adoc

```diff
@@ -99,30 +100,31 @@ in vSphere.
 <9> The password associated with the vSphere user.
 <10> The vSphere datacenter.
 <11> The default vSphere datastore to use.
-<12> Whether to enable or disable FIPS mode. By default, FIPS mode is not enabled. If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
+<12> Optional: The absolute path of an existing folder where the installation program creates the virtual machines, for example, `/<datacenter_name>/vm/<folder_name>/<subfolder_name>`. If you do not provide this value, the installation program creates a folder that is named with the cluster ID in the datacenter virtual machine folder.
```

Comment:
Do we use `cluster id` for reference to infrastructure identifier created by installer for it's infra resources everywhere?

Reply:
You are right. It's "infrastructure ID" in more places in the docs, but it's "cluster ID" in the installer repo.